.github/workflows/build.yml

```diff
@@ -13,7 +13,71 @@ jobs:
         run: meson example/build example && ninja -C example/build
       - uses: actions/upload-artifact@v2
         with:
-          name: ubuntu
+          name: ubuntu-meson
           path: |
             build/
-            example/build/
+            example/build/  build-ubuntu:
+  build-ubuntu-cmake:
+    runs-on: ubuntu-latest
+    steps:
+      - name: install ninja-build
+        run: sudo apt update -y && sudo apt install -y ninja-build
+      - uses: actions/checkout@v2
+      - name: compile library
+        run: cmake -S . -B build && cmake --build build
+      - uses: actions/upload-artifact@v2
+        with:
+          name: ubuntu-cmake
+          path: |
+            build/
+  collect-and-release:
+    needs: [build-ubuntu-meson, build-ubuntu-cmake]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Download artifact ubuntu-meson
+      uses: actions/download-artifact@v3
+      with:
+        name: ubuntu-meson
+        path: ./build-ubuntu-meson
+    - name: Download artifact ubuntu-cmake
+      uses: actions/download-artifact@v3
+      with:
+        name: ubuntu-cmake
+        path: ./build-ubuntu-cmake
+    - name: 'List what is in the working directory'
+      shell: bash
+      run: |
+        ls -la && find .
+   
+    - name: 'Obtain the current version number from the adstool. We need it to create the release.'
+      shell: bash
+      run: |
+        sudo chmod +x ./build-ubuntu-meson/build/adstool
+        echo "GITHUB_ADSLIB_VERSION_NUMBER=$(./build-ubuntu-meson/build/adstool --version)" >> $GITHUB_ENV
```

Comment:
Can't we extract it directly from the debian changelog `dpkg-parsechangelog -SVersion`

Reply:
Okay, done.